sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Remove-SPOSiteManageVersionPolicyJob.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# Remove-SPOSiteManageVersionPolicyJob
+
+## SYNOPSIS
+Stops further processing of site manage version policy job that is in-progress.
+
+> [!NOTE]
+> This feature is currently in preview and may not be available in your tenant.
+
+## SYNTAX
+
+```powershell
+Remove-SPOSiteManageVersionPolicyJob [-Identity] <SpoSitePipeBind> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Stops further processing of site manage version policy job that is in-progress.
```

Comment:
Stops processing of the in-progress manage version policy job for the given site.